added titles to the docs pages (#109)
dappforce/subsocial-docs: docs/develop/sdk/posts/PostWithDetails.md

displayed_sidebar: developSidebar
---

<head>

>   <title>Alternate Methods to Fetch Posts on Subsocial | JS SDK Guide</title>

Reviewer: ```suggestion
  <title>Alternate Methods To Get Posts With The Subsocial JS SDK</title>
```
